xbmc/pvr/epg/EpgInfoTag.cpp

```diff
@@ -245,7 +245,7 @@ float CPVREpgInfoTag::ProgressPercentage() const
   CDateTime::GetCurrentDateTime().GetAsUTCDateTime().GetAsTime(currentTime);
   m_startTime.GetAsTime(startTime);
   m_endTime.GetAsTime(endTime);
-  int iDuration = endTime - startTime > 0 ? endTime - startTime : 3600;
+  time_t iDuration = endTime - startTime > 0 ? endTime - startTime : 3600;
```

Comment:
I really wouldn't use `time_t` for duration. As [cppreference](https://en.cppreference.com/w/cpp/chrono/c/time_t) says, using it for anything else then seconds since 1970-01-01 would be unexpected for most people. I suggest to use [`std::chrono::seconds`](https://en.cppreference.com/w/cpp/chrono/duration) instead. Applies to some other instances in this PR as well.

Reply:
> using it for anything else then seconds since 1970-01-01 would be unexpected for most people

Fair point, I will see what I can do.

Reply:
Please use `std::chrono` so I have less work to do in 18727 :)

Reply:
Even better.